### PR TITLE
new binary test

### PR DESCRIPTION
**Pull request by jasonmotha**
This seems to work. But E2Etests seem to be a bit random.

### CODE REVIEW

**Comment by jasonmotha** [comment]
E2E tests were marking these as failed though they look to be correct / passing.
`/nesi/nobackup/nesi00213/RunFolder/EndToEndTest/tests/tmp_20201104_021135`

**Review by jpa198 — APPROVED** [review]

File: scripts/test_binary.py [hunk 1]
```diff
@@ -65,19 +65,20 @@
...
         acc = bin.acc(stat_name)
-
-        if np.any(acc == 0):
-            if args.verbose:
-                print(
-                    f"The velocities for station {stat_name} contains zero/s, please investigate. This "
-                    f"is most likely due to crashes during HF or BB resulting in no written output."
-                )
-            #  sys.exit(1)  # Commenting this line out until further compatibility investigations - Jason: 20-10-30
+        for comp in acc.T:
```
Comment (shot0829) [inline scripts/test_binary.py:74]: will it not be better to use transpose()? Last time I had issue with this while using newer version of numpy and pandas.

Comment (jasonmotha) [reply scripts/test_binary.py:74]: Apparently they're the same operation? https://stackoverflow.com/questions/4068177/whats-the-difference-of-numpy-ndarray-t-and-numpy-ndarray-transpose-when-self

Comment (shot0829) [reply scripts/test_binary.py:74]: ah, yes, you are correct. I just double checked my commit, it was the returned datatype that had issue, not the method.

**Review by shot0829 — APPROVED** [review]

**Pull request merged**